src/Configuration.php

```diff
@@ -268,6 +268,22 @@ public function setProjectRoot($projectRoot)
         }
     }
 
+    /**
+     * Set the project root regex.
+     *
+     * @param string|null $projectRootRegex the project root path
+     *
+     * @return void
+     */
+    public function setProjectRootRegex($projectRootRegex)
```

Comment:
This method needs adding to the Client class too, following https://github.com/bugsnag/bugsnag-php/commit/cbaf17ac4dbd3cce6f5e16ca67cb251ee19f54dd.